Resources/Prototypes/Traits/categories.yml

```diff
@@ -11,6 +11,13 @@
   id: Quirks
   name: trait-category-quirks
 
+# Begin L5 additions
 - type: traitCategory
   id: Diet
   name: trait-category-diet
+
+- type: traitCategory
+  id: Skills
+  name: trait-category-skills
+  maxTraitPoints: 6
```

Comment:
Opinion: We trust our players to make interesting characters, even if they have all the skills.